CMakeLists.txt

```diff
@@ -379,7 +379,8 @@ if(MSVC)
   default_option(SCCACHE_SUPPORT "Enable sccache support" "SCCACHE_EXECUTABLE")
   message(STATUS "Support for sccache: ${SCCACHE_SUPPORT}")
   if(SCCACHE_SUPPORT)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "sccache")
+    set( CMAKE_C_COMPILER_LAUNCHER "${SCCACHE_EXECUTABLE}" )
+    set( CMAKE_CXX_COMPILER_LAUNCHER "${SCCACHE_EXECUTABLE}" )
```

Comment:
For example, what does this have to do with the sync leader renaming?

Reply:
I merged from main, I think that resolved it

Reply:
Yet, occurrence of such inconsistencies is a warning sign. I am concerned to merge such branches.

Reply:
I'm worried, too. @ywwg any objections against squash-and-merge? Then we can rule out broken commits at least.